cpp/log/cert_test.cc

```diff
@@ -380,11 +380,14 @@ TEST_F(CertTest, Extensions) {
 
   EXPECT_EQ(Cert::TRUE, pre.HasCriticalExtension(cert_trans::NID_ctPoison));
 
-  EXPECT_EQ(Cert::FALSE, leaf.HasBasicConstraintCATrue());
-  EXPECT_EQ(Cert::TRUE, ca.HasBasicConstraintCATrue());
-
-  EXPECT_EQ(Cert::TRUE, ca_pre.HasExtendedKeyUsage(
-                            cert_trans::NID_ctPrecertificateSigning));
+  StatusOr<bool> leaf_ca_status = leaf.HasBasicConstraintCATrue();
+  EXPECT_TRUE(leaf_ca_status.ok() && leaf_ca_status.ValueOrDie() == false);
```

Comment:
Yes wasn't sure about that one. It's easier to read but annoying when tests exit before completion.